Content.Client/Ghost/GhostSystem.cs

```diff
@@ -75,6 +73,15 @@ private void OnGhostRemove(EntityUid uid, GhostComponent component, ComponentRem
             {
                 GhostVisibility = false;
             }
+
+            var eyeManager = IoCManager.Resolve<IEyeManager>().CurrentEye;
+            var lightingManager = IoCManager.Resolve<ILightManager>();
+            if (eyeManager.DrawFov == false || lightingManager.DrawShadows == false|| lightingManager.Enabled == false)
+            {
+                eyeManager.DrawFov = true;
+                lightingManager.DrawShadows = true;
+                lightingManager.Enabled = true;
+            }
```

Comment:
Shouldn't this be done on detach, too?

Reply:
I assumed it would end up being the same since GhostComponent is only used and obtainable by observers and i couldnt think of a case where the component shutdown is not enough or connecting/reconnecting could be abused

Reply:
this should be detach 100%. can imagine you could become ghost -> disconnect but keep client open -> wait for round restart -> connect -> have no fov

Reply:
I tested for that too and didnt seem to work. Still, just in case i added the checks